csgcca: GCC analyzer does not fully support LTO at the moment

lzaoral opened the pull request:
Additionally, due to a bug it uses SSA names of the variables (e.g. current_3
instead of current) in this mode.

See: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=94976

kdudka reviewed (approved):
This fixes a regression from commit 42b14bc4204acc0103af757030b01143744a944a.  Thanks!

lzaoral committed:
csgcca: GCC analyzer does not fully support LTO at the moment

Additionally, due to a bug it uses SSA names of the variables (e.g. current_3
instead of current) in this mode.

Fixes: 42b14bc
See: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=94976
